src/Microsoft.Azure.ServiceBus/ExceptionReceivedEventArgsAction.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Microsoft.Azure.ServiceBus
+{
+    using Microsoft.Azure.ServiceBus.Primitives;
+
+    /// <summary>Action taking place when <see cref="ExceptionReceivedEventArgs"/> is raised for a processed message.</summary>
```

Comment:
Ha-ha, hence [the comment](https://github.com/Azure/azure-service-bus-dotnet/pull/176#issuecomment-306372568): 

> Might need help with XML doc to describe better those actions.